code/modules/research/designs/mechfabricator_designs.dm

```diff
@@ -804,6 +804,15 @@
 	construction_time = 120
 	category = list("Cyborg Upgrade Modules")
 
+/datum/design/borg_upgrade_broomer
+	name = "Cyborg Upgrade (Experimental Push Broom)"
+	id = "borg_upgrade_broomer"
+	build_type = MECHFAB
+	build_path = /obj/item/borg/upgrade/broomer
+	materials = list(/datum/material/iron = 4000, /datum/material/glass = 500)
+	construction_time = 120
+	category = list("Cyborg Upgrade Modules")
```

Comment:
if it literally just adds the broom to their arsenal you can go ahead and make it a default lol

Reply:
i tried to leave it out of the default arsenal as something that should be desired as an upgrade (like mediborg upgrades) because its something that is very useful but the borg can live without

Reply:
broom go brr

Reply:
ok